pymongo/mongo_client.py

```diff
@@ -689,6 +690,14 @@ def _server_property(self, attr_name):
 
         return getattr(server.description, attr_name)
 
+    def watch(self, pipeline=None, full_document='default', resume_after=None,
```

Comment:
Also need a docstring here. You'll also have to add watch to the list of methods in doc/api/pymongo/mongo_client.rst.